ui-admin/src/study/notifications/TriggerBaseForm.tsx

```diff
@@ -0,0 +1,92 @@
+import { Trigger, TriggerActionType, TriggerType } from '@juniper/ui-core'
+import { NavLink } from 'react-router-dom'
+import Select from 'react-select'
+import React from 'react'
+
+const eventTypeOptions = [
+  { label: 'Study Enrollment', value: 'STUDY_ENROLLMENT' },
+  { label: 'Study Consent', value: 'STUDY_CONSENT' },
+  { label: 'Survey Response', value: 'SURVEY_RESPONSE' },
+  { label: 'Kit Sent', value: 'KIT_SENT' },
+  { label: 'Kit Returned', value: 'KIT_RECEIVED' }
+]
+const taskTypeOptions = [
+  { label: 'Survey', value: 'SURVEY' },
+  { label: 'Outreach', value: 'OUTREACH' },
+  { label: 'Consent', value: 'CONSENT' },
+  { label: 'Kit request', value: 'KIT_REQUEST' }
+]
+
+const actionTypeOptions: {label: string, value: TriggerActionType }[] = [
+  { label: 'Notification', value: 'NOTIFICATION' },
+  { label: 'Update task status', value: 'TASK_STATUS_CHANGE' }
+]
+
+const configTypeOptions: { label: string, value: TriggerType}[] = [
+  { label: 'Event', value: 'EVENT' },
+  { label: 'Task reminder', value: 'TASK_REMINDER' },
+  { label: 'Ad hoc', value: 'AD_HOC' }
+]
+
+/** configures the notification type and event/task type */
+export default function TriggerBaseForm({ trigger, setTrigger }:
```

Comment:
i find the UX here very confusing, so I'm looking forward to the trigger list/editing rework!

Reply:
likewise!